File: DashboardX/DashboardX/wwwroot/js/Toastr.js
window.Toastr = (type, message) =>
{
    if (type === "success") {
        toastr.success(message,
            {
                timeOut: 1000
            });
    }
    if (type === "error") {
        toastr.error(message,
            {
                timeOut: 1000
            });
    }
}

File: DashboardX/DashboardX/Helpers/ToastR.cs
using Microsoft.JSInterop;

namespace DashboardX.Helpers;

public class ToastR : BaseRuntime
{
    public ToastR(IJSRuntime runtime) : base(runtime)
    {
    }

    public async Task Success(string successMessage)  => await Invoke("Toastr", "success", successMessage);
    public async Task Error(string successMessage) => await Invoke("Toastr", "error", successMessage);
}
